LayoutTests/imported/w3c/web-platform-tests/css/css-color/parsing/color-valid-relative-color-expected.txt

```diff
@@ -240,19 +237,16 @@ PASS e.style['color'] = "hsla(from rgb(20%, 40%, 60%, 80%) h alpha alpha / alpha
 PASS e.style['color'] = "hsla(from rebeccapurple calc(h) calc(s) calc(l))" should set the property value
 PASS e.style['color'] = "hsla(from rgb(20%, 40%, 60%, 80%) calc(h) calc(s) calc(l) / calc(alpha))" should set the property value
 PASS e.style['color'] = "hsla(from rebeccapurple none none none)" should set the property value
-PASS e.style['color'] = "hsla(from rebeccapurple none none none / none)" should set the property value
+FAIL e.style['color'] = "hsla(from rebeccapurple none none none / none)" should set the property value Colors do not match.
+Actual:   color(srgb 0 0 0 / none)
+Expected: color(srgb 0 0 0 / 0).
+Error: assert_array_approx_equals: Numeric parameters are approximately equal. lengths differ, expected 4 got 3
```

Comment:
Why the regression?

Reply:
The test is incorrect, or at least inconsistent with the computed value version of the test. See https://github.com/w3c/csswg-drafts/issues/10254.